packages/airnode-examples/scripts/choose-integration.ts

```diff
@@ -94,7 +96,31 @@ const chooseIntegration = async (): Promise<IntegrationInfo> => {
   return response as IntegrationInfo;
 };
 
+/**
+ * If git is installed, check if a tag is checked out
+ */
+const checkGitTag = goSync(() => {
+  // skip tag check if git is not present
+  const gitNotFound =
+    spawnSync(`git --version`, {
+      shell: true,
+    }).status !== 0;
+
+  if (gitNotFound) return;
+
+  const gitTag = spawnSync(`git describe --exact-match --tags`, {
+    shell: true,
+  });
+
+  if (gitTag.status !== 0)
+    cliPrint.warning(`Warning:
+    It appears you may not be on a git tag.
+    If you directly downloaded the source code at a specific tag or release, please ignore this warning.
+    Otherwise, please check out a git tag before proceeding (see README for more details).`);
+});
+
 const main = async () => {
+  checkGitTag;
```

Comment:
You forgot to call this :D

Reply:
😅 This comment did identify a bug- the check was happening independently of being called from `main` because of the `goSync` wrapping. Fixed by moving the `goSync` wrapping here within main.